ptop/main.py

```diff
@@ -28,23 +28,127 @@ def main():
                                  simple
                                  blackonwhite
                             ''')
+
+        parser.add_argument('-cpu',
+                            dest='cpu',
```

Comment:
please use some expressive names like `cpuSensorResponseTime` or short forms like `csrt`. Its better for understanding and maintaining.

Reply:
Please if it works now.